tests/io_utils.go

```diff
@@ -117,11 +120,46 @@ func FixErrorDevice(nodeName string) {
 
 }
 
+func executeDeviceMapperOnNode(nodeName string, cmd []string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+
+	// Image that happens to have dmsetup
+	image := fmt.Sprintf("%s/virt-launcher:%s", flags.KubeVirtRepoPrefix, flags.KubeVirtVersionTag)
+	pod := &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "device-mapper-pod-",
+		},
+		Spec: k8sv1.PodSpec{
+			RestartPolicy: k8sv1.RestartPolicyNever,
+			Containers: []k8sv1.Container{
+				{
+					Name:    "launcher",
+					Image:   image,
+					Command: cmd,
+					SecurityContext: &k8sv1.SecurityContext{
+						Privileged: pointer.BoolPtr(true),
```

Comment:
I think you should also add `RunAsUse: 0` as it is not guaranteed root is the only user on the image that is going to used

Reply:
done